python/requirements.txt

```diff
@@ -0,0 +1,4 @@
+setuptools-rust
+setuptools
+wheel
+toml==0.10.2
```

Comment:
Why the exact version pinning here?

Reply:
the rule is that you should always pin your dependencies in requirements.txt, except for `wheel`, because you always need the latest version.
as the readme from setuptools-rust was explicitely not pinning these 3 dependencies, i did the same